week7_review/src/components/edit/EditingArea/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { StyledRoot, WritingForm, SubmitButton } from "./style";
+import { useLocation } from "react-router-dom";
+import axios from "axios";
+import { useNavigate } from "react-router";
+
+function Edit() {
+  const letter = useLocation().state;
```

Comment:
```suggestion
const { name, content, password, hint } = useLocation().state;
```
구조분해 할당 사용해서 써도 될것같아!

Reply:
구조분해할당을 계속 못 활용하는 것 같아 ㅠㅠ letter이라는 객체 내 name이 아니라 name자체만을 받아 구분해서 받을 수 있는건가?